packages/fxa-customs-server/lib/ip_record.js

```diff
@@ -291,6 +310,22 @@ module.exports = function (limits, now) {
       }
     }
 
+    if (actions.isResetPasswordOtpSendingAction(action)) {
+      const otpEmailRateLimited = !!(
+        this.rl &&
+        now() - this.rl < limits.passwordResetOtpEmailRateLimitIntervalMs
+      );
+      if (otpEmailRateLimited || this.shouldBlock()) {
+        return this.retryAfter(limits.passwordResetOtpEmailRateLimitIntervalMs);
+      }
+      this.addPasswordResetOtp();
+      if (this.isOverPasswordResetOtpLimit()) {
+        this.rateLimit();
+        this.os = [];
```

Comment:
Just confirming that we want to reset the ban and not extend it?

Reply:
Yes, because the ip/email will continued to be rate limited.  By extend it do you mean that if the user is currently rate limited we want to return a retry after value that is the most recent timestamp + the rate limit interval?